modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -14,7 +14,7 @@ import styled from '@emotion/styled';
 import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
 import {useModalModel} from './hooks';
 
-export interface ModalOverlayProps extends BoxProps {
+export interface ModalOverlayProps extends Omit<BoxProps, 'children'> {
```

Comment:
Do we have to omit children? Can't we just remove `children` down below and allow BoxProps to pass that children down?

Reply:
yes, I guess it's possible